CONTRIBUTING.md

```diff
@@ -0,0 +1,66 @@
+# Introduction
+
+To begin with, I'd like to welcome you to contribute to the open-source blockchain oracle project. Thank you for your effort, it's what makes this project a viable alternative to commercial solutions.
+
+Please read through these guidelines to get to know how to join the development, submit new issues, pull requests etc... This should increase the effectiveness of time spent by the maintainers.
+
+You're more than welcome to join in any way you see fit, each contribution counts:
+- developing new features
+- improving the code quality by refactoring
+- reviewing pull requests
+- writing documentation
+- suggesting new features
+- reporting and/or fixing bugs
+- discussing potential solutions
+- ...
+
+Having said that, please make sure you don't pull the project your way, against the trends of the community. Strive for open discussion and generic solutions that satisfy the needs of others.
+
+# Ground Rules - Definition of "Done"
+
+Our Definition of "Done" (DoD) includes:
+- close to full unit test coverage
+- code style according to configured linter
+- meaningful commit messages and branch naming
+- ensuring that the CI build passes
+- updating the documentation
+- reviewing the code by at least one maintainer
+
+For issues, code reviews etc. please be respectful and considerate, promote open discussion and provide feedback.
+
+# Getting started
+
+A general guide on contributing code to open source projects can be seen at [contribution-guide.org](https://www.contribution-guide.org).
+
+We've got two branches:
+- `dev` for new features
+- `master` as a stable production branch (and hotfixes)
+
+1. Fork the `dev` branch
+1. Hack away
+1. Make sure you've met the DoD
+1. Create a merge request to the `dev` branch
+1. Wait for code review and apply suggestions
+
+# How to report a bug
```

Comment:
Should we create issue templates and Pull requests templates:

1. https://help.github.com/articles/about-issue-and-pull-request-templates/
1. https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/

Reply:
I know there's such possibility but I think we can add those later on, when the project and community matures a bit. We'll certainly do it when the project gains traction and people won't contribute according to the guidelines.